examples/v1beta2-ex/main.go

```diff
@@ -33,7 +34,7 @@ func main() {
 	}
 	request := kesselv2.ReportResourceRequest{
 		Type:               "host",
-		ReporterType:       "HBI",
+		ReporterType:       "hbi",
```

Comment:
This is one of the rules I relaxed in my PR yesterday, making ReporterType case insensitive. Both examples are accurate.

Reply:
Just an FYI, you don't need to update anything. I'm marking the comment as resolved.